### ISSUE TITLE
NTR: Neisseria gonorrhoeae positive on culture (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8095]
New term request Neisseria gonorrhoeae positive on culture (seidel_190411130740): Gonorrhea; includes culture of blood, eye exudate or drainage

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000902
- label: Positive blood Neisseria gonorrhoeae culture
- parents: [Bacteremia (HP:0031864)](https://hpo.jax.org/app/browse/term/HP:0031864)
- pmid: 
- def: Growth of Neisseria gonorrhoeae in a culture derived from circulating blood.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8095
- synonyms:
